### DISCUSSION TITLE
How do I create a sub-project

### DISCUSSION THREAD

**Discussion by ablink4** [siemens/CtrlppCheck#114] [Q&A] [answered]
I apologize if this is a dumb question.  I've searched in this project's documentation, issues/discussions, the help inside gedi/WinCC OA, and gone through the menu options in the Project Administrator and I cannot figure out how to create a sub-project.  The usage instructions specify that you install WinCCOA_QualityChecks as a subproject, but I don't know how to do that.   Can anyone help me?  I really would appreciate it.

**Comment by jumoog** [comment, marked as answer]
You need to add a new proj_path entry above your project in your project config. Assuming your project name is myproject:

before:

```ini
[general]
pvss_path = "C:/Siemens/Automation/WinCC_OA/3.19"
proj_path = "C:/WinCC_OA_Proj/myproject"
proj_version = "3.19"
useNGA = 1
useSQLite = 1
langs = "de_AT.utf8"
[ui]
defaultFont = "Noto Sans"
```

WinCCOA_QualityChecks added:

```ini
[general]
pvss_path = "C:/Siemens/Automation/WinCC_OA/3.19"
proj_path = "C:/WinCC_OA_Proj/WinCCOA_QualityChecks"
proj_path = "C:/WinCC_OA_Proj/myproject"
proj_version = "3.19"
useNGA = 1
useSQLite = 1
langs = "de_AT.utf8"
[ui]
defaultFont = "Noto Sans"
```

**Reply by ablink4** [reply]
Thank you so much, that helped me add it as a subproject.  I do have some follow-up problems (i.e. it runs the analysis but it appears from the results file cppcheck-result.xml that it checked the sub-project WinCCOA_QualityChecks and not the main project, so I think I'm still doing something wrong.
